omod/src/main/java/org/openmrs/module/adminui/page/controller/systemadmin/accounts/AccountPageController.java

```diff
@@ -203,7 +254,21 @@ public void setModelAttributes(PageModel model, Account account, OtherAccountDat
 		model.addAttribute("allowedLocales", administrationService.getAllowedLocales());
 		List<ProviderRole> providerRoles = providerManagementService.getAllProviderRoles(false);
 		model.addAttribute("providerRoles", providerRoles);
-		
+
+		List<Extension> customPersonEditFragments = appFrameworkService.getExtensionsForCurrentUser("createUser.manageAccountPersonFragment");
```

Comment:
If this is dealing with strictly person attributes, then it should include "PersonAttribute" in its name.

Reply:
Changed